Min-Max_Guide.md

```diff
@@ -276,28 +276,31 @@ An alternate strategy for Spring 1 would be to ignore all of the NPCs and focus
+- Set your UI Scale and Zoom Level to 75%.
+- Immediately chop down nine trees until foraging level 1 (8 1/3 trees needed). We want to get level 1 before we leave to go foraging so that we have a chance to get silver-quality Spring Onions. (Focus on **only** chopping down trees, and begin in the direction of your path to the forest so that some of the weeds you clear are helpful for that.)
+  - Only chop down [Maple Trees](https://stardewvalleywiki.com/Maple_Tree) and [Pine Trees](https://stardewvalleywiki.com/Pine_Tree), not the egg-shaped [Oak Trees](https://stardewvalleywiki.com/Oak_Tree). (We won't get any [Acorns](https://stardewvalleywiki.com/Acorn) for felling an Oak Tree until Spring 2, and acorns are precious.)
   - For the same reason, do not axe/hoe any seeds in the ground until Spring 2, or they will be destroyed.
-  - Do not chop the stumps; only chop a tree until it falls, then move on to the next one. (It yields more wood and more foraging XP to do it this way.)
-  - As soon as a tree starts to fall, walk towards where the wood will spawn and immediately pause the game. This prevents in-game time from passing, but the tree falling animation will still complete. Additionally, all of the wood will automatically be collected by the player while the game is still paused. (You should use this strategy whenever you are chopping trees and not bothering to kill the stumps.)
+  - Do not chop the stumps; only chop a tree until it falls, limiting it to 10 strikes, then move on to the next one. (It yields more wood and more foraging XP to do it this way.) If you lose count, check your skills for foraging level 1 in the menu.
+  - As soon as a tree starts to fall, walk towards where the wood will spawn and immediately pause the game. This approach prevents in-game time from passing, but the tree-falling animation will still be complete. Additionally, the player will automatically collect all of the wood while the game is still paused. (You should use this strategy when chopping trees and not bothering to kill the stumps.)
 - Once foraging level 1, craft a chest, place it next to the house, and empty all inventory except for scythe + axe + pickaxe + hoe.
+- If you chose the four corners or mountain farm, take a screenshot and look at your quarry. Take note of whether there are any copper nodes, geodes, and worms. In particular, additional copper reduces RNG in the mines on day 5. If there is a significant upside in the quarry regarding potential artifacts, ore, and geode gems, take the time now to mine these quickly and hurry your time in the forest afterward. Then, accordingly, later visit the museum and process geodes with Clint.
```

Comment:
i don't think we should bloat the guide with farm-specific tips.
a lot of people choose an intentionally difficult farm (like e.g. beach) in order to make their particular challenge run arbitrarily harder

Reply:
Agreed. I find one of the hardest things in this guide to be day five and reducing the RNG there to be worthwhile. Losing 1-2 of idle time forging at the blacksmith is expensive and potentially pushing out Skull Cavern a day, worse.

Is it sensible to have a discussion on choosing a farm for an optimal playthrough versus a challenge? My guess is that you are recommending the standard but there is some upside to four corners.